packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1937,54 +1939,7 @@ export class ContainerRuntime
 	private readonly entryPoint: LazyPromise<FluidObject>;
 
 	private internalId(maybeAlias: string): string {
-		return this.dataStores.aliases.get(maybeAlias) ?? maybeAlias;
```

Comment:
maybe it's github to blame, but I do not see it being called in this file.

Reply:
@vladsud https://github.com/microsoft/FluidFramework/blob/fb6b1cf25b3df41423a4e65b7f72b249a495908d/packages/runtime/container-runtime/src/containerRuntime.ts#L2568